monitor.py

```diff
@@ -104,6 +105,32 @@ def send_email(subject, message):
     s.quit()
 
 
+def send_slack_notification(subject, message):
```

Comment:
Should factor this out into a separate `slack.py` file that can be used by other parts of the codebase.